index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="Author" content="Shubh M Das">
+    <link rel="stylesheet" href="style.css">
+    <title>portfolio</title>
+</head>
+
+<body>
+   <header>
+        <div class="container">
+            <div id="branding">
+                <a href="#">My Logo</a>
+            </div>
+            <div class="hamburger">
+                <div class="line"></div>
+                <div class="line"></div>
+                <div class="line"></div>
+            </div>
+            </div>
+        </header>
+        
```

Comment:
- [OPTIONAL ] Kindy, Be careful with blank lines and indentation. For readability, add two spaces of indentation. Do not use the tab key. you can check the [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)